src/new-client/src/App/LiveRates/Tiles/ExecutionStatusAlertSwitch/Pending.tsx

```diff
@@ -0,0 +1,42 @@
+import { AdaptiveLoader } from "components/AdaptiveLoader"
+import styled from "styled-components/macro"
+import { OverlayDiv } from "./styled"
+
+const PendingContainer = styled(OverlayDiv)`
+    display: table;
+`
+
+const PendingPopUp = styled("div")`
+  display: table-cell;
+  vertical-align: middle;
+  text-align: center;
+`
+
+const PendingPill = styled("div")`
+  background-color: ${({ theme }) => theme.accents.primary.base};
+  margin-auto;
+  display: inline-block;
+  padding: 1rem 2rem;
+  border-radius: 2rem;
+  font-size: 0.8rem;
+`
+
+const Pending = () => {
```

Comment:
Noticing the spacing (because of course that's what a developer would focus on...).   Prettier doesn't seem to have run on your commit?   Curious, did you commit with `--no-verify`?  If you run `yarn format` do the indents reformat to two spaces?

Reply:
Prettier does not seem to play well with Windows, ignoring my git settings for LF/CRLF

I recloned to WSL2 and pushed up, where the command worked well. Thanks for catching this!